src/renderer/helpers/assetHelper.ts

```diff
@@ -106,13 +107,25 @@ export const assetInERC20Blacklist = (asset: Asset): boolean =>
     O.isSome
   )
 
+/**
+ * Check whether an ERC20 asset is white listed or not
+ */
+export const assetInERC20Whitelist = (asset: Asset): boolean =>
+  FP.pipe(
+    ERC20Whitelist,
+    // add ETH asset
+    A.concat([AssetETH]),
+    A.findFirst((assetInList) => eqAsset.equals(assetInList, asset)),
+    O.isSome
+  )
```

Comment:
Use `addressInList` - Think again if we really need to add `ETH` here - it might be confusing